### PR TITLE
Add WePay and committer list to README.md

### PR DESCRIPTION
**Pull request by criccomini**

### CODE REVIEW

**Comment by criccomini** [comment]
cc @mistercrunch @r39132 @bolkedebruin

**Comment by mistercrunch** [comment]
Should we add the rest of the Airbnb team who have been committing/merging? Or this more as a list where each committer represents a company?

**Comment by criccomini** [comment]
@mistercrunch I think if someone has commit access to this repo, they should be listed.

**Comment by criccomini** [comment]
@mistercrunch if you give me a list, I can add

File: README.md [hunk 1]
```diff
@@ -79,6 +86,7 @@ Currently **officially** using Airflow:
 * [WeTransfer](https://github.com/WeTransfer) [[@jochem](https://github.com/jochem)]
 * Wooga
 * Xoom [[@gepser](https://github.com/gepser) & [@omarvides](https://github.com/omarvides)]
+* [WePay](http://www.wepay.com) [[@criccomini](https://github.com/criccomini) & [@mtagle](https://github.com/mtagle)]
 * Yahoo!
```
Comment (r39132) [inline README.md:90]: You can add bolke under ING to this "currently officially using" list

File: README.md [hunk 2]
```diff
@@ -49,6 +49,13 @@ As the Airflow community grows, we'd like to keep track of who is using
 the platform. Please send a PR with your company name and @githubhandle
 if you may.
 
+Committers:
+
+* [@mistercrunch](https://github.com/mistercrunch) ([Airbnb](http://airbnb.io/))
+* [@r39132](https://github.com/r39132) ([Agari](https://github.com/agaridata))
+* [@criccomini](https://github.com/criccomini) ([WePay](http://www.wepay.com))
+* [@bolkedebruin](https://github.com/bolkedebruin)
```
Comment (r39132) [inline README.md]: ING is the company for bolke. Good to add that if we are adding companies.

**Comment by criccomini** [comment]
@r39132 @mistercrunch @bolkedebruin updated. LMK if it's good to go.

**Comment by bolkedebruin** [comment]
Fine to me

**Pull request merged**